experimental/dds/tree2/src/test/feature-libraries/chunked-forest/chunkTree.spec.ts

```diff
@@ -34,6 +43,24 @@ import {
 } from "./fieldCursorTestUtilities";
 import { polygonTree, testData } from "./uniformChunkTestData";
 
+const builder = new SchemaBuilder("chunkTree");
+const leaf = builder.leaf("leaf", ValueSchema.Serializable);
+const empty = builder.struct("empty", {});
```

Comment:
Is there a way to have reusable schemas? It feels like these shouldn't need to be defined explicitly.

Reply:
this set of schema is specifically selected for this test. While some other tests use very flexible schema (with sequenced of any for example) for their structs since they care about trying different trees and edits, this test suite needs to have some very specific constraints on its schema so it can have some which actually have a constant tree shape for chunking.

Trying to share these schema with other tests would both introduce risks of name collisions, and make adjusting the schema when updating or extending these tests harder as it would impact other tests (more generally, adding unneeded coupling between tests by sharing schema when the tests have specific schema needs) makes maintaince harder.

Generally I'd recommend against sharing schema unless you also share code which uses the schema, or share data which conforms to the schema: otherwise, schema changes are harder, and its harder to know what portion of the schema given code is using.